### PR TITLE
chore!: Removed theme from RadioButton

### PR DESCRIPTION
**Pull request by CarloPalinckx**
### This PR:

**Breaking changes** 🔥
- Removed theme from RadioButton

**Checklist** 🛡
- [x] I have exported my addition from `src/index.ts` (check if not applicable).
- [x] Appropriate tests have been added for my functionality (check if not applicable).
- [x] A designer has seen and approved my changes (tag `@LuukHorsmans` or `@RianneSchaekens` for a design review when applicable).
- [x] I have tested my addition in all supported browsers and for responsiveness (Chrome, Firefox, Safari, Edge, IE11 and mobile browsers).
- [x] Appropriate documentation has been written (check if not applicable).

<details>
  <summary>ℹ️ Conventional commit help</summary>

  - Use `fix: ...` for patches
  - Use `feat: ...` for a minor change
  - Use `feat!: ...` for features with breaking changes
  - Optionally you can use a scope `feat(SomeContainer)!: ...`
  - A breaking change **must** have `BREAKING CHANGE: ...` in the commit message
  - You can use multiple `BREAKING CHANGE: ...` lines in the commit message
  - Use `build`, `chore`, `ci`, `docs`, `perf`, `refactor`, `style`, or `test`  for other types

  More info see full [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/).

  _Tip: make sure the first commit message is the one you want to merge, because that's the one Github picks for squashing._
</details>

### CODE REVIEW

File: packages/components/src/components/RadioButton/index.tsx [hunk 1]
```diff
@@ -3,10 +3,6 @@ import Box from '../Box';
 import Text from '../Text';
 import StyledRadioButton, { StyledRadioButtonSkin, StyledRadioWrapper } from './style';
 
-interface StateType {
-    focus: boolean;
-}
-
 export interface PropsType {
```
Comment (KuiseRapper) [inline packages/components/src/components/RadioButton/index.tsx:6]: PropsType -> Props?

File: packages/components/src/components/RadioButton/style.tsx [hunk 2]
```diff
@@ -1,125 +1,98 @@
+import colors from '../../themes/MosTheme/colors';
+import rgba from '../../utility/rgba';
 import styled from '../../utility/styled';
-import ThemeTools from '../../themes/CustomTheme/ThemeTools';
-import chroma from 'chroma-js';
 
-interface RadioButtonPropsType {
+export interface RadioButtonPropsType {
```
Comment (KuiseRapper) [inline packages/components/src/components/RadioButton/style.tsx:5]: RadioButtonPropsType -> RadioButtonProps?

File: packages/components/src/components/RadioButton/style.tsx [hunk 3]
```diff
@@ -1,125 +1,98 @@
...
     checked: boolean;
 }
 
-interface RadioButtonSkinPropsType {
+export interface RadioButtonSkinPropsType {
```
Comment (KuiseRapper) [inline packages/components/src/components/RadioButton/style.tsx:9]: RadioButtonSkinPropsType -> RadioButtonSkinProps?

**Review by KuiseRapper — CHANGES_REQUESTED** [review]
If u don't agree, let me know. I thought you said without "Type" was better.

File: packages/components/src/components/RadioButton/index.tsx (hunk 1 above)
Comment (CarloPalinckx) [reply packages/components/src/components/RadioButton/index.tsx:6]: Good catch, that is indeed more inline with current naming conventions

**Review by KuiseRapper — APPROVED** [review]

**Pull request merged**